src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -44,6 +44,49 @@ const NodeWrapperPath = "/opt/node-wrapper/"
 const LocalIp = "0.0.0.0"
 const inspectParamVariableName = "ORYX_NODE_INSPECT_PARAM"
 
+// Checks if the application insights needs to be enabled for the current runtime
+func shouldApplicationInsightsBeConfigured() bool {
+	nodeAppInsightsKeyEnv := os.Getenv("APPINSIGHTS_INSTRUMENTATIONKEY")
+	nodeAppInsightsEnabledEnv := os.Getenv("APPLICATIONINSIGHTS_CODELESS_ENABLED")
+
+	if nodeAppInsightsEnabledEnv != "" && nodeAppInsightsKeyEnv != "" {
+		fmt.Printf("Environment Variables for Application Insight's Codeless Configuration exists..\n")
+		return true
+	}
+	return false
+}
+
+// Create application insights loader fiule
+func createApplicationInsightsLoaderFile(appInsightsLoaderFilePath string) {
+	appInsightsLoaderFileContent := `// --------------------------------------------------------------------------------------------
+	// Copyright (c) Microsoft Corporation. All rights reserved.
+	// Licensed under the MIT license.
+	// --------------------------------------------------------------------------------------------
+
+	// Created by Oryx
```

Comment:
nit: since this is duplicated from the build image loader, is it possible to reuse it? I imagine this will be more built-out in the future, specifically to report to the user why attach failed for their app (via UX, as windows currently does with Kudu). It would be ideal to not have to maintain this in 2 places

Reply:
that's a good point .. I'm now thinking to get rid of this logic in build image, because I feel we don't need that there anymore, so this will only stay in runtime